articles/azure/azs-how-create-service-principals.md

```diff
@@ -0,0 +1,97 @@
+---
+title: How to create a service principal for Azure AD 
+description: Give applications access to Azure Stack resources by creating service principals
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to create a service principal for Azure AD 
+toc_fullpath: Users/How To/azs-how-create-service-principals.md
+toc_mdlink: azs-how-create-service-principals.md
+---
+
+# How to Give applications access to Azure Stack resources
```

Comment:
Heading here should match the title in the metadata, so either change here to:

How to create a service principal for Azure AD

Or change in the metadata (title and toc_title) to match this heading.

If you keep this heading -- change Give to give (lowercase g) and possibly use grant instead of give.

Reply:
done